src/main/java/ru/itmo/sd/deadliner2bot/bot/BotImpl.java

```diff
@@ -40,6 +40,8 @@ public void onUpdateReceived(Update update) {
         }
         chatId = update.getMessage().getChatId();
         messageText = Objects.requireNonNullElse(update.getMessage().getText(), "").trim();
+        log.info(update.getMessage().getFrom().getFirstName() + " " + update.getMessage().getFrom().getLastName()
```

Comment:
Logging such information violates users' privacy, we should remove this.

Reply:
Done.